Doc/library/ssl.rst

```diff
@@ -1987,6 +1987,89 @@ to speed up repeated connections from the same clients.
          >>> ssl.create_default_context().verify_mode  # doctest: +SKIP
          <VerifyMode.CERT_REQUIRED: 2>
 
+.. method:: SSLContext.set_psk_client_callback(callback)
```

Comment:
Add documentation on:

- the length limits of the `identity` and `hint` parameters
- how to reject a connection (return a PSK of zero length)